snappy-tools/bin/snappy-shell

```diff
@@ -27,4 +27,19 @@ then
   echo "Using JAVA_ARGS=$JAVA_ARGS"
 fi
 
-exec "$SPARK_HOME"/bin/spark-class $JAVA_ARGS io.snappydata.tools.SnappyUtilLauncher "$@"
+
+function setnewargs() { newargs="$@"; }
+
+if echo $@ | grep -qw "rowstore"; then
+  #using rowstore launcher
+  newargs=
+  for arg in "$@"; do
+    if [[ "$arg" != "rowstore" ]] ; then
+      setnewargs "$newargs" "$arg"
+    fi
```

Comment:
Won't this be better as "-rowstore"? Latter is our normal convention for other options while "start/stop/status" denote actions to be done.